File: 1878-CheckIfArrayIsSortedAndRotated/1878-CheckIfArrayIsSortedAndRotated.java
// Last updated: 8/13/2025, 8:17:06 PM
class Solution {
    public boolean check(int[] nums) {
        int count = 0;
        for(int i=0;i<nums.length;i++){
            int prev = (i+nums.length-1)%nums.length;
            if(nums[prev]>nums[i]){
                count++;
            }
        }
        if(count<=1){
            return true;
        }
        return false;
    }
}

File: 1878-CheckIfArrayIsSortedAndRotated/1878-CheckIfArrayIsSortedAndRotated.py
# Last updated: 8/13/2025, 8:16:55 PM
class Solution(object):
    def check(self, nums):
        count = 0
        for i in range(len(nums) -1) :
            if(nums[i] > nums[i+1]) :
                count +=1
        if(nums[len(nums)-1] > nums[0]) :
            count += 1
        if(count <= 1) :
            return True
        return False